superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -176,6 +179,11 @@ def _inference_step(self, precision):
                     self._model(sample)
                     end = self._timer()
                     curr_step += 1
+                    if self._args.log_every_steps:
+                        if curr_step % self._args.log_every_steps == 0:
+                            print(
+                                f'{self._name} - {precision.value}: step {curr_step}, step time {(end - start) * 1000}'
+                            )
```

Comment:
Could we put these code into function, and call function directily?

Reply:
sure, updated